### ISSUE TITLE
[flaky test] Test_Run_Positive_VolumeFSResizeControllerAttachEnabled

### ISSUE THREAD

**Issue by BenTheElder** [kubernetes/kubernetes#99815]
#### Which jobs are flaking:

pull-kubernetes-unit

#### Which test(s) are flaking:

`k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler`: `Test_Run_Positive_VolumeFSResizeControllerAttachEnabled/expand-fs-volume_with_in-use_error`

#### Testgrid link:

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/99810/pull-kubernetes-unit/1367610862453395456

(PR with no go code changes, should not have failed any unit tests)

#### Reason for failure:

```
=== RUN   Test_Run_Positive_VolumeFSResizeControllerAttachEnabled
=== CONT  Test_Run_Positive_VolumeFSResizeControllerAttachEnabled
    testing.go:1093: race detected during execution of test
--- FAIL: Test_Run_Positive_VolumeFSResizeControllerAttachEnabled (0.52s)
```

#### Anything else we need to know:
- links to go.k8s.io/triage appreciated
- links to specific failures in spyglass appreciated

/sig storage

**Comment by BenTheElder** [comment]
```
WARNING: DATA RACE
Write at 0x00c0007140c0 by goroutine 99:
  runtime.mapassign_faststr()
      /usr/local/go/src/runtime/map_faststr.go:202 +0x0
  k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler.Test_Run_Positive_VolumeFSResizeControllerAttachEnabled.func1()
      /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/kubelet/volumemanager/reconciler/reconciler_test.go:1151 +0x1a9e
  testing.tRunner()
      /usr/local/go/src/testing/testing.go:1194 +0x202
```

**Comment by pacoxu** [comment]
/assign
tried to fix it in #100215
